tgui/assets/tgui.css

```diff
@@ -1 +1 @@
2px;background-color:#272727;border:none}body.solgov .hidden{display:none}body.solgov .bar .barText,body.solgov span.button{color:#fff;font-size:12px;font-weight:400;font-style:normal;text-decoration:none}body.solgov .bold{font-weight:700}body.solgov .italic{font-style:italic}body.solgov [unselectable=on]{-webkit-user-select:none;-moz-user-select:none;-ms-user-select:none;user-select:none}body.solgov div[data-tooltip],body.solgov span[data-tooltip]{position:relative}body.solgov div[data-tooltip]:after,body.solgov span[data-tooltip]:after{position:absolute;display:block;z-index:2;width:250px;padding:10px;-ms-transform:translateX(-50%);transform:translateX(-50%);visibility:hidden;opacity:0;-ms-filter:"progid:DXImageTransform.Microsoft.Alpha(Opacity=0)";color:#fff;pointer-events:none;white-space:normal;text-align:left;content:attr(data-tooltip);transition:all .5s;border:1px solid #272727;background-color:#363636}body.solgov div[data-tooltip]:hover:after,body.solgov span[data-tooltip]:hover:after{visibility:visible;opacity:1;-ms-filter:"progid:DXImageTransform.Microsoft.Alpha(Opacity=100)"}body.solgov div[data-tooltip].tooltip-top:after,body.solgov span[data-tooltip].tooltip-top:after{bottom:100%;left:50%;-ms-transform:translateX(-50%) translateY(8px);transform:translateX(-50%) translateY(8px)}body.solgov div[data-tooltip].tooltip-top:hover:after,body.solgov span[data-tooltip].tooltip-top:hover:after{-ms-transform:translateX(-50%) translateY(-8px);transform:translateX(-50%) translateY(-8px)}body.solgov div[data-tooltip].tooltip-bottom:after,body.solgov span[data-tooltip].tooltip-bottom:after{top:100%;left:50%;-ms-transform:translateX(-50%) translateY(-8px);transform:translateX(-50%) translateY(-8px)}body.solgov div[data-tooltip].tooltip-bottom:hover:after,body.solgov span[data-tooltip].tooltip-bottom:hover:after{-ms-transform:translateX(-50%) translateY(8px);transform:translateX(-50%) translateY(8px)}body.solgov div[data-tooltip].tooltip-left:after,body.solgov span[data-tooltip].tooltip-left:after{top:50%;right:100%;-ms-transform:translateX(8px) translateY(-50%);transform:translateX(8px) translateY(-50%)}body.solgov div[data-tooltip].tooltip-left:hover:after,body.solgov span[data-tooltip].tooltip-left:hover:after{-ms-transform:translateX(-8px) translateY(-50%);transform:translateX(-8px) translateY(-50%)}body.solgov div[data-tooltip].tooltip-right:after,body.solgov span[data-tooltip].tooltip-right:after{top:50%;left:100%;-ms-transform:translateX(-8px) translateY(-50%);transform:translateX(-8px) translateY(-50%)}body.solgov div[data-tooltip].tooltip-right:hover:after,body.solgov span[data-tooltip].tooltip-right:hover:after{-ms-transform:translateX(8px) translateY(-50%);transform:translateX(8px) translateY(-50%)}body.solgov .bar{display:inline-block;position:relative;vertical-align:middle;width:100%;height:20px;line-height:17px;padding:1px;border:1px solid #000;background:#272727}body.solgov .bar .barText{position:absolute;top:0;right:3px}body.solgov .b
```

Comment:
Oh uhhh.... Im learning this as i go. That makes sense, who would write this code.